### PR TITLE
docs(demo): make yaml links relative

### LINKED ISSUE #2840: port-forward script failed for bookbuyer
**Bug description**:
Following https://github.com/openservicemesh/osm/blob/release-v0.8/docs/content/docs/install/manual_demo/_index.md, after deploying the app yamls, running`./scripts/port-forward-all.sh` failed for bookbuyer:

```console
kubectl port-forward -n bookbuyer bookbuyer-79bbd489d8-j52jr 8080:14001 
Forwarding from 127.0.0.1:8080 -> 14001
Forwarding from [::1]:8080 -> 14001
Handling connection for 8080
Handling connection for 8080
E0312 13:20:14.085451   61171 portforward.go:400] an error occurred forwarding 8080 -> 14001: error forwarding port 14001 to pod c52b91ecb68a1a33500d3f097c4f22298521abc05d4d048de2d4e33b2728bb92, uid : exit status 1: 2021/03/12 21:20:14 socat[8027] E connect(5, AF=2 127.0.0.1:14001, 16): Connection refused
```
Seems the reason for this failure is that as of commit 65f4f8545daf the readme is pointing to the yamls in main and main does not have the new image tags that carry the port change for bookbuyer.
 
**Affected area (please mark with X where applicable)**:
- Install                    [ ]
- SMI Traffic Access Policy  [ ]
- SMI Traffic Specs Policy   [ ]
- SMI Traffic Split Policy   [ ]
- Permissive Traffic Policy  [ ]
- Ingress                    [ ]
- Egress                     [ ]
- Envoy Control Plane        [ ]
- CLI Tool                   [ ]
- Metrics                    [ ]
- Certificate Management     [ ]
- Sidecar Injection          [ ]
- Logging                    [ ]
- Debugging                  [ ]
- Tests                      [ ]
- Demo                       [ ]
- CI System                  [ ]

**Expected behavior**:

**Steps to reproduce the bug (as precisely as possible)**:

**How was OSM installed?**:

**Anything else we need to know?**:

**Environment**:
- OSM version (use `osm version`):
- Kubernetes version (use `kubectl version`):
- Size of cluster (number of worker nodes in the cluster):
- Others:

### PR DESCRIPTION
**Pull request by ritazh**
Signed-off-by: Rita Zhang <rita.z.zhang@gmail.com>

**Description**:
Currently the demo yamls are pointing to the main branch, which could be incompatible with a particular release. This PR updates the yamls so that they are relative path rather than pointing to the yamls in main.
NOTE: this commit should be cherry picked to release-0.8 branch

Fixes #2840 

**Affected area**:

- New Functionality      [ ]
- Documentation          [ ]
- Install                [ ]
- Control Plane          [ ]
- CLI Tool               [ ]
- Certificate Management [ ]
- Networking             [ ]
- Metrics                [ ]
- SMI Policy             [ ]
- Security               [ ]
- Tests                  [ ]
- CI System              [ ]
- Performance            [ ]
- Other                  [ ]

Please answer the following questions with yes/no.

- Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?
No

### CODE REVIEW

File: docs/content/docs/install/manual_demo/_index.md [hunk 1]
```diff
@@ -71,10 +71,10 @@ osm namespace add bookstore bookbuyer bookthief bookwarehouse
 Deploy `bookbuyer`, `bookthief`, `bookstore`, `bookwarehouse` applications:
```
Comment (ksubrmnn) [inline docs/content/docs/install/manual_demo/_index.md:71]: can you add a note somewhere that all these yamls should be applied from the root of the repo?

File: docs/content/docs/install/manual_demo/_index.md [hunk 2]
```diff
@@ -71,10 +71,10 @@ osm namespace add bookstore bookbuyer bookthief bookwarehouse
...
 
 ```bash
-kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/main/docs/example/manifests/apps/bookbuyer.yaml
-kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/main/docs/example/manifests/apps/bookthief.yaml
-kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/main/docs/example/manifests/apps/bookstore.yaml
-kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/main/docs/example/manifests/apps/bookwarehouse.yaml
+kubectl apply -f docs/example/manifests/apps/bookbuyer.yaml
+kubectl apply -f docs/example/manifests/apps/bookthief.yaml
+kubectl apply -f docs/example/manifests/apps/bookstore.yaml
+kubectl apply -f docs/example/manifests/apps/bookwarehouse.yaml
```
Comment (shashankram) [inline docs/content/docs/install/manual_demo/_index.md:77]: These links were updated to absolute URLs so the user wouldn't need to check out the repo to run the demo. 
PR https://github.com/openservicemesh/osm/pull/2733 updated it for this purpose. /cc @michelleN

Comment (ritazh) [reply docs/content/docs/install/manual_demo/_index.md:77]: I see. We do have this in the `Configure Prerequisites` section of this page right now
```
- Have a local clone of the OSM GitHub Repo
  - `git clone https://github.com/openservicemesh/osm.git`
  - `cd osm`
 ```

Comment (shashankram) [reply docs/content/docs/install/manual_demo/_index.md:77]: Cloning the repo should be optional if one uses released binaries available directly on the release page.

**Review by draychev — APPROVED** [review]
This looks good to me.

I am going to rewrite this demo with #2845 and make this the main and only demo. (Will remove the 'auto' demo and drop 'manual' from this with #2842)

I'd like to take this discussion to #2845 - my plan is to indicate which version of OSM the demo is meant for, and also capture these YAMLs in the doc itself - without pointing to links.
This would a) not have links out of the doc and b) make it easy to scrol, copy/paste, scroll, copy/paste (no need to navigate away from the doc and come back and find where you were) which I believe makes it a better demo / user experience.

**Comment by ritazh** [comment]
Closing this in favor of https://github.com/openservicemesh/osm/pull/2845

**Pull request closed without merging**